src/JuMP.jl

```diff
@@ -143,6 +143,17 @@ function setObjectiveSense(m::Model, newSense::Symbol)
 end
 setObjective(m::Model, something::Any) =
     error("in setObjective: needs three arguments: model, objective sense (:Max or :Min), and expression.")
+
+# # Hacky little fallbacks so that arrays of length 1 act as scalars for adding constraints and setting objective
```

Comment:
No need to leave in commented code. But do we give a nice message now when a vector is given for the objective?